storybook-example/.storybook/main.ts

```diff
@@ -1,14 +1,25 @@
-import type { StorybookConfig } from "@storybook/web-components-webpack5";
+import { StorybookConfig } from "@storybook/react-webpack5";
+import { refs } from "./refs";
 
 const path = require("path");
 const SRC_PATH = path.join(__dirname, "../../sign-in-with-esignet/src");
```

Comment:
can we update the variable names to more be specific

Reply:
which variable name you want to change @anshulv1401 ?